test/core/crawlers/http_crawler.test.ts

```diff
@@ -0,0 +1,153 @@
+import { HttpCrawler } from '@crawlee/http';
+import type { AddressInfo } from 'net';
+import http from 'http';
```

Comment:
lets use the `node:` prefix, ideally we should find some lint rule for it

Reply:
Yes I'm constantly forgetting about this